src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  removeTodo: (todoId: number) => void;
+  isLoading: boolean;
+  selectedId: number | null;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  removeTodo,
+  isLoading,
+  selectedId,
+}) => {
+  return (
+    <div
+      data-cy="Todo"
+      className="todo"
+    >
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+        />
+      </label>
+
+      <span data-cy="TodoTitle" className="todo__title">
+        {todo.title}
+      </span>
+      <button
+        type="button"
+        className="todo__remove"
+        data-cy="TodoDeleteButton"
+        onClick={() => removeTodo(todo.id)}
+      >
+        ×
+      </button>
+
+      {
+        selectedId === todo.id && (
+          <div
```

Comment:
```suggestion
      {selected && (
          <div
```

const selected = selectedId === todo.id